### PR TITLE
add logging, change to non-json log observer

### PR DESCRIPTION
**Pull request by shintasmith**
Two minor changes:
* change the log observer from the json one to the simple ones (less noise, makes it easier to debug issues)
* add log message to print response code when POSTing calls to blueflood

### CODE REVIEW

**Comment by shintasmith** [comment]
@drenalin23 @usnavi @ChandraAddala @VinnyQ @izrik please review, thx

**Comment by drenalin23** [comment]
Another set of eyes would be good but lgtm.

File: carbonforwarderlogging/forwarder_log_observer.py [hunk 1]
```diff
@@ -10,6 +11,14 @@
 
 def get_log_observer():
     f = logfile.LogFile("carbon_forwarder.log", log_dir, rotateLength=log_rotate_length, maxRotatedFiles=max_rotated_log_files)
+    observer = FileLogObserver(f)
+    filterer = FilteringLogObserver(observer.emit,
+        [LogLevelFilterPredicate(
+            LogLevel.levelWithName(log_level))])
+    return filterer
+
+def get_json_log_observer():
```
Comment (ChandraAddala) [inline carbonforwarderlogging/forwarder_log_observer.py]: How calls this method?

Comment (shintasmith) [inline carbonforwarderlogging/forwarder_log_observer.py]: It is called this way:
```
    twistd --logger carbonforwarderlogging.forwarder_log_observer.get_json_log_observer blueflood-forward
```
That's the documented way to define custom logging in the twistd world.

**Comment by ChandraAddala** [comment]
Looks good.

**Pull request merged**